tests/slow/src/test/scala/tests/feature/ImportMissingSymbolCrossLspSuite.scala

```diff
@@ -39,7 +41,7 @@ class ImportMissingSymbolCrossLspSuite
             |""".stripMargin,
         s"""|${ImportMissingSymbol.title("A", "example")}
             |${CreateNewSymbol.title("A")}""".stripMargin,
-        Nil,
+        List(ImportMissingSymbolQuickFix.kind),
```

Comment:
I wouldn't change this, since the behaviour in VS Code is to send empty by default. I'd rather make sure we are not showing more than we want.

We do try to duplicate VS Code behaviour as much as we can here.